src/config.ts

```diff
@@ -17,7 +18,7 @@ export function getSecure<T>(key: string, workspaceState: vscode.Memento): T|
 // Like get(), but won't implicitly load settings from workspace config.
 // If there is workspace config, prompts the user and caches the decision.
 export async function getSecureOrPrompt<T>(
-    key: string, workspaceState: vscode.Memento): Promise<T> {
+    key: string, workspaceState: vscode.Memento): Promise<T|undefined> {
```

Comment:
(again here I think we should assert the result rather than returning a maybe-result)

Reply:
I'll fix that.